internal/importers/base/importer.go

```diff
@@ -251,6 +251,12 @@ func (i *Importer) ImportPackages(packages []ImportedPackage, defaultConstraintF
 			pc.Constraint = gps.Any()
 		}
 
+		// Skip constraints that are empty (no branch, version or source)
+		if gps.IsAny(pc.Constraint) && pc.Ident.Source == "" {
+			i.Logger.Printf("  Skipping import of %v because its constraint is empty.\n", pc.Ident)
```

Comment:
I think this may get noisy for some importers. Let's put the log statement behind a `i.Verbose` check.